sysbuild/Kconfig.bt_fast_pair

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
Add empty line with `#` at the top and bottom

Reply:
Fixed